Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -420,10 +420,13 @@ private void Start()
                 constraintsManager = gameObject.EnsureComponent<ConstraintManager>();
             }
 
-            if (!AllowFarManipulation && GetComponent<NearInteractionGrabbable>() == null)
+            // Get child objects with NearInteractionGrabbable attached
+            var children = GetComponentsInChildren<NearInteractionGrabbable>();
+
+            if (children.Length == 0)
             {
-                Debug.LogError($"AllowFarManipulation is false and a NearInteractionGrabbable component is not attached to {gameObject.name}. " +
-                " To enable near interaction support, attach a NearInteractionGrabbable component to this object.");
+                Debug.LogWarning($"Near interactions are not enabled for {gameObject.name}. To enable near interactions, add a " +
```

Comment:
I'm assuming the UX team reviewed and decided that the 80% use case will include far and that the warning should be shown by default? If so, this looks correct. Wondering if this should be log level **Warning** or log level **Info**. **Info** would still allow the message to be shown but would not be considered a **Warning**. Just an option. I'm perfectly OK either way.

Reply:
I think this would be better as an info message rather than a warning as not having near interaction support does not impact the core functionality of Object Manipulator.